### DISCUSSION TITLE
September 7th, 2022

### DISCUSSION THREAD

**Discussion by LukasBuecherl** [SynBioDex/SBOL-Editors#22] [Editors meetings]
# Participants
 * Chris Myers
 * James Scott-Brown
 * Lukas Buecherl

# Agenda

## Website (James S-B)
* Nothing to report
* Finishing migration of co.mbine.org 
* Lukas reaching out to Pedro about assigned tasks

## SEPs / Github issues (Prashant)
* No updates

## Communications (Gonzalo)
* No updates

## Report on other SBOL meetings (Tom)
* No updates (Tom will join next week)

**Comment by Gonza10V** [comment]
I've got the slots for our breakout rooms at COMBINE. Please let me know if these times works for you:

- Thursday, 09:00, SEP055 
- Thursday, 14:00, SBOL 3.1
-  Friday, 14:00 SBOL Examples

If there are no problems with the times we will confirm the assigned slots.
